src/raw/money/icon_16.svg

```diff
@@ -0,0 +1,7 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path d="M2.75 6.00024C3.16421 6.00024 3.5 5.66446 3.5 5.25024C3.5 4.83603 3.16421 4.50024 2.75 4.50024C2.33579 4.50024 2 4.83603 2 5.25024C2 5.66446 2.33579 6.00024 2.75 6.00024Z" fill="#47474F"/>
```

Comment:
Wrong fill color. This hex is not part of the list that will be replaced with "currentColor", hence this will not work. Applies to all fill colors in this PR.

Reply:
#47474F is now added to the list ✅